docs/notes/2.22.x.md

```diff
@@ -18,6 +18,12 @@ Keep reading to see the details and what's also included.
 
 ### Backends
 
+#### SQL
+
+A new experimental `SQL` backend was added along with the [sqlfluff
+linter](https://www.pantsbuild.org/2.20/reference/subsystems/sqlfluff). See
+docs [here](https://www.pantsbuild.org/2.20/docs/sql).
```

Comment:
I guess these links ought to be towards the 2.22 version of the docs?

Reply:
yep, you're right https://github.com/pantsbuild/pants/pull/20854/commits/fb82f9350c2d1d770ee9c42cb8fcdda95722dbf4